### PR TITLE
Update Tizen-docs for certificate manager and tizen-core changes

### PR DESCRIPTION
**Pull request by abhishek-kp1**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/tizen-studio/common-tools/certificate-registration.md [hunk 1]
```diff
@@ -42,7 +42,10 @@ You can create a new certificate profile with the Certificate Manager:
 
       ![Author information](./media/cert_reg_author_info.png)
 
-   2. You can use the default Tizen distributor certificate or another distributor certificate if you have one. In general, the default Tizen distributor certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level of the distributor certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
+   2. You can use the default Tizen distributor certificate or another distributor certificate if you have one. In general, the default Tizen distributor certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level and version of the distributor certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
```
Comment (omar-srbd) [inline docs/application/tizen-studio/common-tools/certificate-registration.md:45]:
```suggestion
   2. You can use the default Tizen distributor certificate or another distributor certificate if you have one. In general, the default Tizen Distributor Certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level and version of the Distributor Certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
```

File: docs/application/tizen-studio/common-tools/certificate-registration.md [hunk 2]
```diff
@@ -42,7 +42,10 @@ You can create a new certificate profile with the Certificate Manager:
...
+
+      * For Tizen-8.0 devices or the higher versions you can select version as new(default).
+      * For Tizen-7.0 devices or the lower versions you can select version as old.
```
Comment (omar-srbd) [inline docs/application/tizen-studio/common-tools/certificate-registration.md:48]:
```suggestion
      * For Tizen 8.0 devices or higher versions you can select version as new (default).
      * For Tizen 7.0 devices or lower versions you can select version as old. 
```

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 3]
```diff
@@ -1188,21 +1188,20 @@ To package and install rpk on the target, use the following commands:
 
 ## Import project to Tizen-Core format
 
-The projects created in Tizen Studio, Visual Studio, and other IDEs need to be converted to Tizen-Core format, before building and running the projects in Tizen-Core. 
-Import command in Tizen-Core is used to convert and create the required files to build the project in Tizen-Core.
-Initialize the workspace with the required type as mentioned in the section [Initialize the Workspace](#initialize-the-workspace). After the workspace is initialized, use the following command to import the project:
+The projects created in Tizen-Studio, Visual Studio, and other IDEs need to be converted to Tizen-Core format, before building and running the projects in Tizen-Core.
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:1191]:
```suggestion
The projects created in Tizen Studio, Visual Studio, and other IDEs need to be converted to Tizen-Core format, before building and running the projects in Tizen-Core. 

```

File: docs/application/tizen-studio/tizen-core/tizen-core-cli.md [hunk 4]
```diff
@@ -1188,21 +1188,20 @@ To package and install rpk on the target, use the following commands:
 
 ## Import project to Tizen-Core format
 
-The projects created in Tizen Studio, Visual Studio, and other IDEs need to be converted to Tizen-Core format, before building and running the projects in Tizen-Core. 
-Import command in Tizen-Core is used to convert and create the required files to build the project in Tizen-Core.
-Initialize the workspace with the required type as mentioned in the section [Initialize the Workspace](#initialize-the-workspace). After the workspace is initialized, use the following command to import the project:
+The projects created in Tizen-Studio, Visual Studio, and other IDEs need to be converted to Tizen-Core format, before building and running the projects in Tizen-Core. 
+Init command in Tizen-Core is used to convert and create the required files to build the project in Tizen-Core as Import functionality is accomplished internally through the
+Init command.
```
Comment (omar-srbd) [inline docs/application/tizen-studio/tizen-core/tizen-core-cli.md:1193]:
```suggestion
init command in Tizen-Core is used to convert and create the required files to build the project in Tizen-Core as import functionality is accomplished internally through the init command.
```

**Review by omar-srbd — COMMENTED** [review]
Hello @abhishek-kp1, I have reviewed the files in this PR and left some suggestions for you.

Please check and incorporate them accordingly.

Thank you.

**Comment by omar-srbd** [comment]
Hello @abhishek-kp1, this PR has been inactive for a while.
Please check the suggestions and incorporate them.

Thank you.

**Comment by abhishek-kp1** [comment]
Hello @omar-srbd, I have made the necessary changes. Can you please review it again ?

Thank you.

File: docs/application/tizen-studio/common-tools/certificate-registration.md [hunk 5]
```diff
@@ -42,7 +42,10 @@ You can create a new certificate profile with the Certificate Manager:
 
       ![Author information](./media/cert_reg_author_info.png)
 
-   2. You can use the default Tizen distributor certificate or another distributor certificate if you have one. In general, the default Tizen distributor certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level of the distributor certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
+   2. You can use the default Tizen distributor certificate or another distributor certificate if you have one. In general, the default Tizen Distributor Certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level and version of the Distributor Certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
```
Comment (omar-srbd) [inline docs/application/tizen-studio/common-tools/certificate-registration.md:45]:
```suggestion
   2. You can use the default Tizen Distributor Certificate or another distributor certificate if you have one. In general, the default Tizen Distributor Certificate is used and you do not need to modify the distributor certificates. You can also select the privilege level and version of the Distributor Certificate (in [native](../../native/tutorials/details/sec-privileges.md) and [Web](../../web/tutorials/sec-privileges.md) applications).
```

**Review by omar-srbd — COMMENTED** [review]
Hello @abhishek-kp1, I have gone through the changes and found one issue to have gone unaddressed.

Please check the suggestion and incorporate the change.

Thank you.

File: docs/application/tizen-studio/common-tools/certificate-registration.md (hunk 5 above)
Comment (abhishek-kp1) [reply docs/application/tizen-studio/common-tools/certificate-registration.md]: changes resolved. Thank you.

**Comment by abhishek-kp1** [comment]
Hi @omar-srbd, I have made the necessary changes, You can look into it.

Thank you.

**Pull request merged**